src/main/webapp/mentors.js

```diff
@@ -79,7 +140,66 @@ function createMentorElement(mentor) {
 }
 
 /**
- * Creates a github profile button element from username.
+ * Creates an email for mentor card.
+ * @param {String} email
+ * @return {HTMLElement} mentorEmail
+ */
+function createEmail(email) {
+  const mentorEmail = document.createElement('a');
+  mentorEmail.href = 'mailto: ' + email;
+  mentorEmail.innerText = 'Send Email Introduction';
+  return mentorEmail;
+}
+
+/**
+ * Creates a location for mentor card.
+ * @param {String} location
+ * @return {HTMLElement} mentorLocation
+ */
+function createLocation(location) {
+  const mentorLocation = document.createElement('h6');
+  mentorLocation.className = 'card-subtitle text-muted text-center mb-1';
+  mentorLocation.innerText = location;
+  return mentorLocation;
+}
+
+/**
+ * Creates an image for mentor card.
+ * @param {String} img
+ * @return {HTMLElement} mentorImg
+ */
```

Comment:
I think the function names here could be more specific, for example `createLocationHeader`.